views/results/_table.html

```diff
@@ -1,18 +1,20 @@
-<table>
-    <thead>
-    <tr>
-        <th>Created At</th>
-        <th>Keyword</th>
-        <th>Status</th>
-    </tr>
-    </thead>
-    <tbody>
-        {{ range $result := .results }}
+{{ range $results := .resultSets }}
+    <table class="table table-results">
+        <thead class="table-headers">
```

Comment:
Lets use singular class name

Reply:
✅  fixed in [4c10b49](https://github.com/carryall/go-google-scraper-challenge/pull/66/commits/4c10b490b9cf075bde20e98c4da06b0592c387a3)